packages/react-components/react-card/package.json

```
@@ -52,9 +52,7 @@
   },
   "beachball": {
     "disallowedChangeTypes": [
-      "major",
-      "minor",
-      "patch"
+      "major"
```

You should also disallow `prerelease`:

```suggestion
      "major",
      "prerelease"
```

Once you do, you'll probably need to regenerate your change file.